dataland-batch-manager/src/test/kotlin/org/dataland/batchmanager/service/GleifApiAccessorTest.kt

```diff
@@ -67,6 +67,13 @@ class GleifApiAccessorTest {
         mockFileUtils.verify({ FileUtils.copyURLToFile(any(), any()) }, times(1))
     }
 
+    @Test
+    fun `test if download of full golden copy relationships works fine under the right conditions`() {
```

Comment:
what does "under the right conditions mean"? is it then necessary a test to prove wrong conditions?